artifactory/utils/npm/pack.go

```diff
@@ -27,8 +27,7 @@ func createPackCmdConfig(executablePath string, splitFlags []string) *npmutils.N
 	}
 }
 
-func getPackageFileNameFromOutput(output string) (string, error) {
+func getPackageFileNameFromOutput(output string) []string {
 	output = strings.TrimSpace(output)
-	lines := strings.Split(output, "\n")
-	return strings.TrimSpace(lines[len(lines)-1]), nil
+	return strings.Split(output, "\n")
```

Comment:
will work on windows?

Reply:
Looks like from the CLI test that it is working on windows 👍